src/main/java/Event.java

```diff
@@ -0,0 +1,18 @@
+public class Event extends Task {
+
+    protected String at;
```

Comment:
Perhaps naming of the variables could be done a little better, Something like `dueAt`?

Reply:
Thanks for spotting this @darshan152 !